contents/stats.md

```diff
@@ -0,0 +1,430 @@
+# Statistics {#sec:stats}
+
+This content was created in order to awaken the reader to the importance of statistics for science and knowledge generation.
+Our idea is to present the concepts in the way we would like to have been provided when students about to be initiated into science.
+Our approach is to simplify the concepts as much as possible without losing their essence.
+**In this chapter, we will cover**:
+
+* what is **statistics**
+* the difference between **descriptive statistics** and **inferential statistics**
+* measures of **central tendencies**
+* measures of **dispersion**
+* measures of **dependence**
+* **probability distributions**
+* statistical **visualization**
+
+Statistics is important because it is a **tool to make sense of data**.
+With the abundant availability of data, we are often overwhelmed by numbers.
+Statistics offers a way to comprehend, summarize and infer information from data.
+We believe that every data scientist should have a basic understanding of statistics and how to perform simple statistical operations.
```

Comment:
```suggestion
We believe that every data scientist needs to have a basic understanding of statistics and how to perform simple statistical operations.
```
otherwise I agree with @rikhuijzer 's suggestion.

Reply:
Ok let's go with "needs".